### DISCUSSION TITLE
Cropping Batch of files for BMW DISPLAY

### DISCUSSION THREAD

**Discussion by tuggseven** [staxrip/staxrip#903] [Q&A]
Hey Guys, 
i crop some Youtube Videos for my BMW Display so i had a decend Crop (1920x720)
Since i had read that StaxRip uses GPU im Testing out the Configurations for this but cant do it for Batch of files...

I have about 40 Files (â 3-4min) to Crop in this format automaticly but cant do it. When i Load it in they had all their own Configurations..

Can anybody tell me how to Config the Export Correctly for Batching?  My Settings, NVIDIA H.264 , MP4, 1920x720 Crop Center, Demux Audio "None"

Thx for your Help =))

**Comment by jessielw** [comment]
Create a template with the desired crop. Disable auto crop and save the template. Then batch youe files with that template.

**Reply by tuggseven** [reply]
Yes i think the same, but How to Disable the AutoCrop on Import Action?
Anything i do dont change the Import Action .. -.-

**Reply by tuggseven** [reply]
It's change the ratio but i dont want that its autamaticly.  Staticly on 1920x720 that its just fit the display completely..

**Reply by jessielw** [reply]
It's in the menu options. Uncheck auto crop.
